hip-hcc/PKGBUILD

```diff
@@ -1,16 +1,16 @@
 # Maintainer: acxz <akashpatel2008 at yahoo dot com>
 pkgname=hip-hcc
 pkgver=3.3.0
-pkgrel=5
+pkgrel=6
 pkgdesc="Heterogeneous Interface for Portability ROCm"
 arch=('x86_64')
 url="https://github.com/ROCm-Developer-Tools/HIP"
```

Comment:
Can we change the url to https://rocmdocs.amd.com/en/latest/Installation_Guide/HIP.html

Reply:
Done.